app/src/main/java/org/p2p/wallet/infrastructure/network/data/ServerException.kt

```diff
@@ -8,7 +8,8 @@ class ServerException(
     val errorCode: ErrorCode,
     fullMessage: String,
     private val errorMessage: String?,
-    val jsonErrorBody: JsonObject? = null
+    val jsonErrorBody: JsonObject? = null,
+    val domainErrorType: RpcTransactionError? = null
```

Comment:
doesn't break work with this class
still exception, but with additional nullable trycatched field